setup.py

```diff
@@ -25,7 +25,8 @@
 
 setup(
     name="scorpio",
-    version="0.0.1-rc1",
+    version="0.0.1-rc2",
```

Comment:
Already updated in #228 - rebase, thanks.

Reply:
Yes. Another cherry pick from another PR. I'll fix that. good suggestion.